server/controllers/list.js

```diff
@@ -304,7 +304,7 @@ const getListData = (req, resp) => {
         cohortName,
         description,
         isArchived,
-        isGuest: !isMember(list, userId),
+        isGuest: cohortId && !isMember(cohort, userId),
```

Comment:
what is this is a private list not within the cohort??
Does it recognize that the user is a guest??